### PR TITLE
Removes the filters

### PR DESCRIPTION
**Pull request by SireTurret**
## Changelog
:cl: Turret
del: Dumbspeech filters have been removed. No more braindamage from saying Barnyard or Enya.
/:cl:

## About The Pull Request

Removes the shitty ass filter. Bye!

## Why It's Good For The Game

The filter looks for some shit like "nya" in other words, like barnyard. Absoutely gay.

### CODE REVIEW

**Comment by karmaisblackandbluepilled** [comment]
+1
it was funny for a while but then i saw someone ahelp about getting a seizure and dying because he said "wiz has barnyard curse"

**Comment by Manly-Dwarf** [comment]
rouge probably does not to be removed
I see Vivadel decided to add some extras to it, like "hola", for no reason at all.

**Comment by Manly-Dwarf** [comment]
also I edited the changelog description since yours was terrible

File: hippiestation/code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -22,26 +22,6 @@
 	if (!.)
 		return
 
-	if(findtext(message, "rouge") || findtext(message, "owo") || findtext(message, "uwu") || findtext(message, "nya") || findtext(message, "hola") || findtext(message, "hue"))
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/mob/living/say.dm:25]: there is no reason to delete all of this. just keep rouge and delete the additions

File: hippiestation/code/modules/mob/living/say.dm [hunk 2]
```diff
@@ -22,26 +22,6 @@
 	if (!.)
 		return
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/mob/living/say.dm]:
```suggestion
if(findtext(message, "rouge"))
		var/mob/living/carbon/human/H = src
		to_chat(src, "<span class='warning'><b>You feel like a fucking idiot.</b></span>")
		playsound_local(src, 'hippiestation/sound/effects/whistlefail.ogg', 50, 0)	//Rip
		if(istype(src, /mob/living/carbon/human/))
			var/trauma_type = pickweight(list(BRAIN_TRAUMA_MILD = 100,BRAIN_TRAUMA_SEVERE = 30,BRAIN_TRAUMA_MAGIC = 10,BRAIN_TRAUMA_SPECIAL = 1))
			var/trauma_resistance
			switch(trauma_type)
				if(BRAIN_TRAUMA_MILD)
					trauma_resistance = TRAUMA_RESILIENCE_BASIC
				if(BRAIN_TRAUMA_SEVERE)
					trauma_resistance = TRAUMA_RESILIENCE_SURGERY
				if(BRAIN_TRAUMA_MAGIC)
					trauma_resistance = TRAUMA_RESILIENCE_MAGIC
				if(BRAIN_TRAUMA_SPECIAL)
					trauma_resistance = TRAUMA_RESILIENCE_BASIC
			H.adjustBrainLoss(20)
			H.gain_trauma_type(trauma_type, trauma_resistance)
```
very basic change

**Comment by SireTurret** [comment]
updated

**Comment by SireTurret** [comment]
thanks KBB

**Review by ghost — DISMISSED** [review]

**Comment by MaterialisticThings** [comment]
+1 but only if it removes the dumb ones

**Comment by yoyobatty** [comment]
https://ci.appveyor.com/project/yoyobatty/hippiestation/builds/28564455
`
loading hippiestation.dme
hippiestation\code\modules\mob\living\say.dm:27:error: bad argument definition
hippiestation\code\modules\mob\living\say.dm:28:error: bad argument definition
hippiestation\code\modules\mob\living\say.dm:28:error: bad argument definition
hippiestation\code\modules\mob\living\say.dm:28:error: bad argument definition
hippiestation\code\modules\mob\living\say.dm:29:error: inconsistent indentation
hippiestation\code\modules\mob\living\say.dm:30:error: inconsistent indentation
hippiestation\code\modules\mob\living\say.dm:31:error: inconsistent indentation
hippiestation\code\modules\mob\living\say.dm:33:error: inconsistent indentation
hippiestation\code\modules\mob\living\say.dm:35:error: inconsistent indentation
hippiestation\code\modules\mob\living\say.dm:37:error: inconsistent indentation
hippiestation\code\modules\mob\living\say.dm:39:error: inconsistent indentation
hippiestation\code\modules\mob\living\say.dm:40:error: inconsistent indentation
hippiestation\code\modules\mob\living\say.dm:41:error: inconsistent indentation
hippiestation\code\modules\mob\living\say.dm:41:error: bad argument definition
hippiestation\code\modules\mob\living\say.dm:42:error: inconsistent indentation
hippiestation.dmb - 15 errors, 0 warnings (11/2/19 10:52 am)
Total time: 0:34
Command exited with code -1
`
ruh ruh

**Comment by SireTurret** [comment]
what

**Comment by ghost** [comment]
you cant say coworker without oWo

**Comment by hovershift01** [comment]
Uwu?

**Comment by waterwater1212** [comment]
-1

**Comment by karmaisblackandbluepilled** [comment]
+1 filters are awfully coded.

**Comment by yoyobatty** [comment]
@SireTurret 

You gotta make sure your build tools are passing:
image

If you click continuous-integration it compiles the code in a test environment

**Comment by SireTurret** [comment]
it works bros..

**Comment by yoyobatty** [comment]
Yep, if it doesn't click into it and the 2nd test of the 3 tells you the exact error, same with appveyor which is even faster but not quite as robust.

**Review by ghost — APPROVED** [review]
what is there to approve?

**Comment by ghost** [comment]
looks like it works, merg time

**Pull request merged**